compose.production.yaml

```diff
@@ -241,6 +241,7 @@ services:
       - /opt/olsystem/etc/infobase.yml:/infobase.yml
     networks:
       - webnet
+      - dbnet
```

Comment:
@cdrini, do you think this change is needed? I'm not convinced it's necessary

Reply:
I double checked this and I believe you are correct, @mekarpeles. Locally I had defined `affiliate_server` in my local `openlibrary.yml` to use the internal Docker hostname/service name of the affiliate server, but it appears that in production `affiliate_server` uses `ol-home0` rather than the Docker hostname for the affiliate server.

I've removed this change.